onnxruntime/test/providers/cpu/math/element_wise_ops_test.cc

```diff
@@ -183,6 +183,22 @@ TEST(MathOpTest, DimWithZeroHandling) {
   run(test5);
 }
 
+TEST(MathOpTest, Add_int8) {
```

Comment:
These tests are good. But, they are for C++ only. Would you please also add tests to ONNX so that a broader community will be benefited? 

Like this:
https://github.com/onnx/onnx/blob/main/onnx/backend/test/case/node/mul.py#L35

Our pipelines run all ONNX node tests